docs/features/basic-pod.md

```diff
@@ -0,0 +1,25 @@
+# Quickstart - Basic Pod Scheduling
+This Quickstart guide will walk you through how to schedule a pod using the Gödel Scheduling System.
+
+## Local Cluster Bootstrap & Installation
+
+Please refer to [Cluster Set-Up Guide](./docs/features/kind-cluster-setup.md) for creating a local KIND cluster installed with Gödel.
```

Comment:
1. Set-Up -> Setup
2. the link is broken as the relative path is not correct

Reply:
Fixed both